flowsa/methods/flowbysectormethods/Water_state_2015_m1.yaml

```diff
@@ -1,4 +1,5 @@
 target_sector_level: NAICS_4
+target_subset_sector_level: {NAICS_6: ['1133', '1125']}
```

Comment:
Are these true targets here or just sample? A small comment might help to clarify why those are being targeted.